iam/src/utils/logging.ts

```diff
@@ -0,0 +1,18 @@
+import winston, { createLogger, format, transports } from "winston";
```

Comment:
is it possible to also silence all `console.log`s (at least in production)? `brightid_sdk` is logging every axios request they make 😒